extensions/amp-jwplayer/1.0/amp-jwplayer.css

```diff
@@ -0,0 +1,3 @@
+amp-jwplayer {
+  /** Component styles */
```

Comment:
Feel free to take from brightcove - extensions/amp-brightcove/1.0/amp-brightcove.css

Reply:
forgot about these styles - added